domenetjenester/behandling-revurdering/src/main/java/no/nav/ung/sak/behandling/revurdering/sats/OpprettRevurderingHøySatsBatchTask.java

```diff
@@ -52,17 +56,23 @@ public void doTask(ProsessTaskData prosessTaskData) {
 
         Set<Map.Entry<Fagsak, LocalDate>> fagsakerTilRevurdering = satsEndringRepository.hentFagsakerMedBrukereSomFyller25ÅrFraDato(datoForKjøring).entrySet();
 
+
         List<ProsessTaskData> prosessTaskerTilRevurdering = utledProsessTaskerForRevurdering(fagsakerTilRevurdering);
 
         taskGruppeTilRevurderinger.addNesteParallell(prosessTaskerTilRevurdering);
         prosessTaskTjeneste.lagre(taskGruppeTilRevurderinger);
     }
 
-    static List<ProsessTaskData> utledProsessTaskerForRevurdering(Set<Map.Entry<Fagsak, LocalDate>> entries) {
+    List<ProsessTaskData> utledProsessTaskerForRevurdering(Set<Map.Entry<Fagsak, LocalDate>> entries) {
         return entries
             .stream()
             .map(fagsakerTilVurdering -> {
                 Fagsak fagsak = fagsakerTilVurdering.getKey();
+                var prosesstaskerForFagsak = fagsakProsessTaskRepository.finnAlleForAngittSøk(fagsak.getId(), null, null, List.of(ProsessTaskStatus.KLAR, ProsessTaskStatus.VETO, ProsessTaskStatus.FEILET), true, null, null);
+                if (prosesstaskerForFagsak.stream().anyMatch(task -> task.getTaskType().equals(OpprettRevurderingEllerOpprettDiffTask.TASKNAME))) {
```

Comment:
Bør vi også sjekke at det er samme perioder og behandlingsårsak?

Reply:
Nei, det vil fortsatt feile sjølv om det er ulike perioder. Grunnen til at vi gjer denne sjekken er ikkje for idempotens, men for å unngå ein situasjon som fører til at vi manuelt må patche saker. Dersom saken fortsatt ikkje er på høg sats etter neste revurdering vil denne opprett ein ny revurdering neste dag.